src/anyio/__init__.py

```diff
@@ -89,4 +89,18 @@
     if getattr(__value, "__module__", "").startswith("anyio."):
         __value.__module__ = __name__
 
-del __value
+
+def __getattr__(attr: str):  # type: ignore[no-untyped-def]
```

Comment:
This is the only thing it returns, might as well use it as a return type hint?
```suggestion
def __getattr__(attr: str) -> type[BrokenWorkerInterpreter]:
```